app/scenes/Document/components/Document.js

```diff
@@ -383,28 +381,36 @@ class DocumentScene extends React.Component<Props> {
             >
               {document.isTemplate && !readOnly && (
                 <Notice muted>
-                  You’re editing a template. Highlight some text and use the{" "}
-                  <PlaceholderIcon color="currentColor" /> control to add
-                  placeholders that can be filled out when creating new
-                  documents from this template.
+                  <Trans>
+                    You’re editing a template. Highlight some text and use the{" "}
+                    <PlaceholderIcon color="currentColor" /> control to add
+                    placeholders that can be filled out when creating new
+                    documents from this template.
+                  </Trans>
                 </Notice>
               )}
               {document.archivedAt && !document.deletedAt && (
                 <Notice muted>
-                  Archived by {document.updatedBy.name}{" "}
-                  <Time dateTime={document.archivedAt} /> ago
+                  <Trans>
+                    Archived by {document.updatedBy.name}{" "}
+                    <Time dateTime={document.archivedAt} /> ago
+                  </Trans>
```

Comment:
For the translation framework to correctly interpolate dynamic values they must be in the format of `{{ thing }}` (with spaces for consistency). This comment applies to several of the changes in this file – see here as a working example:

https://github.com/outline/outline/blob/e845652cb8f9295c13bfe1c2d2324654b9b3dd77/app/menus/NewChildDocumentMenu.js#L34-L38

I think something like this would work:

```suggestion
                  <Trans
                    defaults="Archived by {{ userName }} <time> ago"
                    values={{ userName: document.updatedBy.name }}
                    components={{ time: <Time dateTime={document.archivedAt} /> }}
                  />
```